recipes-bsp/atf/atf-poplar_git.bb

```diff
@@ -0,0 +1,41 @@
+DESCRIPTION = "ARM Trusted Firmware Poplar"
```

Comment:
if the whole point of ATF recipe is to be used by l-loader, I'm inclined to do the same as HiKey960's UEFI: https://github.com/96boards/meta-96boards/blob/master/recipes-bsp/uefi/edk2-hikey960_git.bb

i.e. a recipe that will build the bootloader bits that we need.